src/material-experimental/mdc-chips/chip.ts

```diff
@@ -179,7 +179,7 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
   get value(): any {
     return this._value !== undefined
       ? this._value
-      : this._elementRef.nativeElement.textContent;
+      : this._elementRef.nativeElement.querySelector('.mdc-chip__text').textContent.trim();
```

Comment:
Can we resolve this once up-front, rather than querying every time? This could become a performance issue if it ends up being used somewhere in a view.